instance-segmentation/README.md

```diff
@@ -0,0 +1,105 @@
+# COCO Instance Segmentation
+
+[![Open in Layer](https://development.layer.co/assets/badge.svg)](https://app.layer.ai/layer/instance-segmentation/) [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/layerai/examples/blob/main/instance-segmentation/notebooks/demo.ipynb) [![Layer Examples Github](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com/layerai/examples/tree/main/instance-segmentation)
+
+![Segments AI](https://segments.ai/blog/assets/images/panoptic-datasets/2d.png)
+
+In this project, we are going to train an **instance segmentation** model with a [COCO Dataset](https://cocodataset.org/#home) from [Segments AI](https://segments.ai/). 
+
+Segments AI is a platform for computer vision experts to manage, label and improve datasets which supports image, point cloud and test labeling in various formats including the COCO Instance we've used in this project. 
+
+
+## How to make predictions
+
+You can load and use the trained model from this project easily. 
+First make sure you have the latest version of Layer and Pytorch:
+```
+!pip install layer -q
+!pip install torch
+```
+
+Then, you can fetch the model from Layer and make predictions from an image
+
+```python
+import layer
+import torch
+
+
+my_model = layer.get_model("layer/instance-segmentation/models/object_detector:4.2").get_train()
+my_model.eval()
+
+img = ... # Your tensor image
```

Comment:
I think you can update this to include the image/tensor here https://github.com/layerai/examples/blob/mecevit/instance-segmentation/instance-segmentation/notebooks/demo.ipynb